dotnet-authserver/src/TeacherIdentity.AuthServer/Program.cs

```diff
@@ -224,6 +224,7 @@ public static async Task Main(string[] args)
             options.AddPolicy(
                 AuthorizationPolicies.Authenticated,
                 policy => policy
+                    .AddAuthenticationSchemes("Delegated")
```

Comment:
This is the fix to ensure we get a `UserSignedInEvent` emitted when hitting first the account pages.